### ISSUE TITLE
Mistake in Jupyter Server

### ISSUE THREAD

**Issue by forestbat** [VeinsOfTheEarth/RivGraph#80]
I want to run some demo of rivgraph in my jupyter server,however when I try to launch the code, it crashed like this:

>Traceback (most recent call last)
Input In [2], in <cell line: 1>()
----> 1 from rivgraph.classes import river
      2 import matplotlib.pyplot as plt
      3 import os

ModuleNotFoundError: No module named 'rivgraph'

And before this I have installed rivgraph's conda environment:

> (rivgraph) jupyterhub@vm-jupyterhub-server:~/graphs$ conda env list
rivgraph             *  /home/jupyterhub/.conda/envs/rivgraph
base                     /opt/conda
db-base                  /opt/conda/envs/db-base
gis-base                 /opt/conda/envs/gis-base
lf                       /opt/conda/envs/lf
ml-base                  /opt/conda/envs/ml-base
r-base                   /opt/conda/envs/r-base

So how to solve this problem?

**Comment by jonschwenk** [comment]
Hi forestbat, I believe our installation documentation might be outdated. Basically, the `enviornment.yml` file doesn't specify to install the `rivgraph` package--however, it does create a virtual environment with the required dependencies. So you just need to install `rivgraph`. You have two options:

1) install from a clone of the Github repo, instructions [here](https://veinsoftheearth.github.io/RivGraph/install/index.html#installation-from-source).

2) activate your rivgraph environment and run `conda install rivgraph -c jschwenk`.

Option 1 will give you the "cutting-edge" version of the code, and Option 2 will give you the latest stable release.

**Comment by forestbat** [comment]
> Hi forestbat, I believe our installation documentation might be outdated. Basically, the `enviornment.yml` file doesn't specify to install the `rivgraph` package--however, it does create a virtual environment with the required dependencies. So you just need to install `rivgraph`. You have two options:
> 
>     1. install from a clone of the Github repo, instructions [here](https://veinsoftheearth.github.io/RivGraph/install/index.html#installation-from-source).
> 
>     2. activate your rivgraph environment and run `conda install rivgraph -c jschwenk`.
> 
> 
> Option 1 will give you the "cutting-edge" version of the code, and Option 2 will give you the latest stable release.

I have run "conda install rivgraph -c jschwenk" under rivgraph env,however it still told me "ModuleNotFoundError: No module named 'rivgraph'",and I can find rivgraph in `conda list`.

**Comment by elbeejay** [comment]
@forestbat I've just tested package installation from source in Google's co-laboratory cloud environment (which may be similar to your jupyter server), my working notebook can be viewed [here](https://colab.research.google.com/drive/1gxaqWHErwx1FQn3-EpUXz3MFLprSxGcp?usp=sharing). 

For completeness, the installation steps I used are as follows:

1. Cloning the repository: `git clone https://github.com/jonschwenk/RivGraph.git`
2. Change directories into cloned folder: `cd RivGraph`
3. Install dependencies (contents of the `environment.yml` file): `mamba install geopandas scikit-image opencv 'networkx<2.7' 'shapely=1.7.1' matplotlib fastdtw loguru -y`
4. Install the package from source: `python setup.py install`
